include/cuco/static_map.cuh

```diff
@@ -1424,3 +1863,4 @@ class static_map {
 }  // namespace cuco
 
 #include <cuco/detail/static_map.inl>
```

Comment:
Is GH UI weird or are we including both `<cuco/detail/static_map.inl>` and `<cuco/detail/static_map/static_map.inl>`?

Reply:
`<cuco/detail/static_map.inl>` is the legacy code and `<cuco/detail/static_map/static_map.inl>` is where the new experimental map located. `<cuco/detail/static_map.inl>` to be removed once we get rid of the legacy map.